charts/kubewarden-controller/templates/_helpers.tpl

```diff
@@ -99,10 +99,15 @@ Create the name of the service account to use for kubewarden-controller
 
 {{- define "audit-scanner.command" -}}
 - /audit-scanner
+- --kubewarden-namespace
+- {{ .Release.Namespace }}
 - --loglevel
-- info
+- {{- .Values.auditScanner.logLevel }}
 - --extra-ca
-- "/pki/policy-server-root-ca-pem"
+- {{- .Values.auditScanner.policyServerCa }}
+{{- if .Values.auditScanner.enableJsonReport }}
+- --print
+{{- end }}
```

Comment:
Is `--print` the cli option used to force the output format to be json? If that's the case we should change audti-scanner because this is rather counterintuitive